### PR TITLE
Update user be

### PR DESCRIPTION
**Pull request by GarrettCottrell**
## What's this PR do?
- This PR creates a mutation, resolver and tests for the updating user profile functionality

## Relevant tickets
- #56 

## Where should the reviewer start?
- The reviewers can start in the Crate/code/api/src/modules/user/mutations.js file and look at the userUpdate mutation. From there, reviewer can go to the Crate/code/api/src/modules/user/resolvers.js file and look at the "update profile" resolver. From there, the reviewer can go to the Crate/code/api/src/modules/user/tests/mutations.test.js file and look at the full test file which was created to test the update profile mutation. 

## Any background context you want to provide?
- This functionality allows the user table in the database to be updated by calling the updateuser mutation

### CODE REVIEW

File: code/api/src/modules/user/tests/mutations.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    )
+  })
+
+  afterAll(done => {
+    connection.close();
+    done();
+  });
+
+  it('can update user profile', async (done) => {
+    var email = 'test@test.com'
```
Comment (saraho1123) [inline code/api/src/modules/user/tests/mutations.test.js:31]: I am not seeing address or image in here. Should that be a part of this? @GarrettCottrell  @jdcarey128  @the-color-bliu

**Review by saraho1123 — COMMENTED** [review]
I have one question in the `code/api/src/modules/user/tests/mutations.test.js` It may not be a big deal though!

**Review by kncaputo — APPROVED** [review]
Looks good. I see users will now be able to add update their address, description, image and email.

File: code/api/src/modules/user/tests/mutations.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    var description = 'test description'
+    var id = 2
+ const response = await request(server)
+      .post('/')
+      .send({
+        query: `mutation{userUpdate(id: ${id}, email: "${email}", description: "${description}" ) {id email description image shippingAddress}}`})
```
Comment (CalebCyphers) [inline code/api/src/modules/user/tests/mutations.test.js:37]: Loving the structure of this!

**Review by CalebCyphers — APPROVED** [review]

**Review by the-color-bliu — APPROVED** [review]

**Pull request merged**